sunpy/net/vso/__init__.py

```diff
@@ -1,4 +1,5 @@
 # for exposure to from sunpy.net.vso import *
-from sunpy.net.vso.vso import QueryResponse, VSOClient
+from sunpy.net.vso.table_response import VSOQueryResponseTable
+from sunpy.net.vso.vso import VSOClient
```

Comment:
The change in what's being imported here needs a breaking changelog entry

Reply:
Have continued to import QueryResponse.